src/vs/workbench/common/editor/editorStacksModel.ts

```diff
@@ -944,6 +944,16 @@ export class EditorStacksModel implements IEditorStacksModel {
 		return this._groups.indexOf(group);
 	}
 
+	public findGroup(editorInput: EditorInput, activeOnly: Boolean): EditorGroup {
+		const found = this.groups.filter(group =>
```

Comment:
@bpasero The code before assumed that we prefer active editor and groups over inactive as it made sure to first check the active ones.